src/components/chart/statistics-chart.ts

```diff
@@ -245,6 +248,32 @@ export class StatisticsChart extends LitElement {
       },
       // @ts-expect-error
       locale: numberFormatToLocale(this.hass.locale),
+      onClick: (e: any) => {
+        if (
+          !this.clickForMoreInfo ||
+          !(e.native instanceof MouseEvent) ||
+          (e.native instanceof PointerEvent && e.native.pointerType !== "mouse")
+        ) {
+          return;
+        }
+
+        const chart = e.chart;
+
+        const points = chart.getElementsAtEventForMode(
+          e,
+          "nearest",
+          { intersect: true },
+          true
+        );
+
+        if (points.length) {
+          const firstPoint = points[0];
+          fireEvent(this, "hass-more-info", {
+            entityId: this._statisticIds[firstPoint.datasetIndex],
```

Comment:
We should check if the statistics id is an entity id first

Reply:
Use `isExternalStatistic`:
https://github.com/home-assistant/frontend/blob/8f67ddf968eba518fb7d091bcc5cca6a37b0e98e/src/data/recorder.ts#L325

If it is an external statistic, we should not try to open the more info.

Reply:
tbh, didn't know yet that there is something like "external statistics" ... so hopefully c8f9e32 covers what you mean?